generative_data_prep/__main__.py

```diff
@@ -297,7 +298,7 @@ def get_categories(categories_path: str):
 
 if __name__ == "__main__":
     logger = logging.getLogger("generative_data_prep_logger")
-    logging.config.fileConfig("configs/logger.conf")
+    logging.config.fileConfig(get_config_file_path())
```

Comment:
please move `configs` inside `generative_data_prep`; placing it outside causes all sorts of issues